RELEASING.md

```diff
@@ -14,5 +14,22 @@ kpt-functions-catalog repo.
 1. Click `Publish release` button.
 1. Send an announcement email in the [kpt users google group].
 
+## Updating function docs
+
+After creating a release, the docs for the function should be updated to reflect
+the latest patch version. A script has been created to automate this process.
+
+1. Run the script:
+```shell
+# Check out latest version of the make target from master
+git fetch upstream && git checkout upstream/master
+# Run the make target
+# e.g. RELEASE_BRANCH=set-namespace/v0.3 make update-function-docs
+RELEASE_BRANCH=<RELEASE_BRANCH> make update-function-docs
+```
+2. The script will generate a new commit in your local repository which updates
+the docs for the provided function release. Push this commit and create a pull
+request targeted at the release branch.
```

Comment:
Can you highlight these two questions:
1. running this make rule will switch branches, can you tell the source-branch (HEAD detached at upstream/master) to the local release branch (${RELEASE_BRANCH}. qq: shall user delete the local release-branch or make the local-release branch be up to date with the remote upstream RELEASE_BRANCH?
2. The example command user shall run

Reply:
Yeah it seems there is an assumption that the local release branch is up-to-date with remote. I added a step to ensure that this is the case. Alternatively the script itself could probably handle checking out the remote, although the usage of the local branch is useful for testing